config.ini

```diff
@@ -1,6 +1,6 @@
 [postgres]
```

Comment:
I would remove config, because each of us can have different connection strings

Reply:
Agreed, removed